rs_control/scripts/plot_joints.py

```diff
@@ -0,0 +1,334 @@
+#!/usr/bin/env python
```

Comment:
I beleve that the control node should contain only from control files. Move the plotting files to the other node

Reply:
The file consists only subscribers.